integration-test/trino-it/shutdown.sh

```diff
@@ -5,4 +5,13 @@
 #
 cd "$(dirname "$0")"
 
+# change the hive container's logs directory permission
+docker exec trino-ci-hive chown -R `id -u`:`id -g` /tmp/root
+docker exec trino-ci-hive chown -R `id -u`:`id -g` /usr/local/hadoop/logs
+
+# for trace file permission
+ls -l ../build/trino-ci-container-log
+ls -l ../build/trino-ci-container-log/hive
+ls -l ../build/trino-ci-container-log/hdfs
```

Comment:
Is it possible to remove this part of the code so that it is only executed once in launch?

Reply:
If it's only executed at startup, it's possible that files generated later still have 'root' as their owner. A better solution here would be to modify the Docker configuration to have it start Docker using the user that runs the pipeline.